docs/StardustDocs/topics/readSqlDatabases.md

```diff
@@ -451,6 +447,40 @@ val schema = DataFrame.getSchemaForSqlQuery(connection, "SELECT * FROM Users WHE
 connection.close()
 ```
 
+### Extension functions for schema retrieval from an SQL query or SQL table
+
+The same example, rewritten with the extension function:
+
+```kotlin
+import java.sql.Connection
+import java.sql.DriverManager
+
+val connection = DriverManager.getConnection("URL_TO_CONNECT_DATABASE")
+
+val schema = connection.getDataFrameSchema("SELECT * FROM Users WHERE age > 35")
+
+connection.close()
+```
+**Connection.getDataFrameSchema(sqlQueryOrTableName: String): DataFrameSchema**
+
+Retrieves the schema of an SQL query result or the SQL table using the provided database configuration.
```

Comment:
~the~ SQL table. Should be "an", just as "an SQL query result", but you can leave it out.